src/Data/Textile/LifeCycle.elm

```diff
@@ -153,6 +152,6 @@ updateSteps labels update_ lifeCycle =
     labels |> List.foldl (\label -> updateStep label update_) lifeCycle
 
 
-encode : Definitions -> LifeCycle -> Encode.Value
-encode definitions =
-    Encode.array (Step.encode definitions)
+encode : LifeCycle -> Encode.Value
+encode =
+    Encode.array Step.encode
```

Comment:
Do we really need this proxy, or should we now rather use `Encode.array Step.encode` directly? 🤔